### DISCUSSION TITLE
Preview of updated pi-base viewer app

### DISCUSSION THREAD

**Discussion by StevenClontz** [pi-base/data#321] [Show and tell]
The app hosted at https://topology.pi-base.org hasn't been updated in some time, but a major refactor has been going on and can be previewed at

https://topology.pages.dev

Please poke around there and leave feedback here as you have time over the next week or so. If appropriate, you can also open an issue at https://github.com/pi-base/web/issues and add the `prerelease` milestone (but cross-post that issue here too).

**Comment by StevenClontz** [comment]
Tip: I need to write it up (that is, how to reproduce the bug), but I've found that I've occasionally broken the site. If this happens to you, go to the [Advanced](https://topology.pages.dev/dev) tab, then hit `Reset`.

**Comment by StevenClontz** [comment]
On the advanced tab you can change the branch you're previewing (very handy for reviewing PRs for instance). But right now this is quite brittle and breaks the site every so often: https://github.com/pi-base/web/issues/65

**Comment by prabau** [comment]
I like that the top level Spaces tab now brings up the list of spaces, parallel to the one for Properties or for Theorems.  We did not have that before.

On the list of properties for a particular space (example: https://topology.pages.dev/spaces/S000004/properties), it used to be that clicking on a property would link to the page for that property and clicking in the Value column would bring an explanation for the property in that particular space.  Now both link the the explanation for the particular space only.  I think the previous behavior was more useful, as it allows to easily open the general info for a property in a separate tab in the browser for example.

On the same page, the filter at the top to search for properties matching a pattern does not work anymore.

In the explanation page for a particular property for a particular space when it is deduced (example: https://topology.pages.dev/spaces/S000004/properties/P000056), the links from the Value column don't work for me.  I left click on them and nothing happens (both Firefox and Edge browsers).  But if I right click on them and choose "open link in new tab", it does work. (actually, as mentioned by @ccaruvana below, it manages to pop up a new window but with the wrong contents)

And somehow things get into a bad state if I click too many things.  So there may be more but that's all I can say for now.

**Comment by ccaruvana** [comment]
In addition to what @prabau said about explanations for properties, see the difference between https://topology.pages.dev/spaces/S000004/properties/P000001 and https://topology.pi-base.org/spaces/S000004/properties/P000001
Even though they are linking to the same file, presumably, the new viewer just reiterates the definition of the space and does not provide the explanation for the why the provide does/does not hold.

**Comment by lyengulalp** [comment]
When clicking on particular properties or spaces, the pages are not loading to the correct thing. For example, from the Theorems section of Metrizable (https://topology.pages.dev/properties/P000053/theorems) I can see Second Countable as something that Metrizable implies. Clicking on Second Countable in the "then" column brings me to https://topology.pages.dev/properties/P000027, which you would expect to be the Second Countable property page. Instead, it still displays Metrizable and pops back to the spaces tab. Refreshing the page seems to fix it. Also, opening in a new tab seems to work fine.

I don't see this same behavior on the old site.

**Comment by StevenClontz** [comment]
Thanks for the stress test everyone. Keep feedback coming if you find other issues, but I'll keep you posted as James and I have time to bugfix some. (I'm slammed for the next two weeks running workshops though.)
